Gemfile

```diff
@@ -13,4 +13,5 @@ group :development, :test do
   gem 'paperclip'
   gem 'rake'
   gem 'rake-rspec'
+  gem 'pry'
```

Comment:
can you remove pry? I assume this was for debugging.

Reply:
sure I thought it could be useful for others